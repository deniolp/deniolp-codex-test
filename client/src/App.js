import React from 'react';

function App(props) {
const {data} = props;
console.log(data);
  return (
    <div>
      <h1>Hi friends!</h1>
    </div>
  );
}

export default App;
