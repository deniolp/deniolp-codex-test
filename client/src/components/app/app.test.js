import React from 'react';
import renderer from 'react-test-renderer';

import App from './app';

describe(`App`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
      <App data={[`C 44 16, L 1 2 12 2`]}/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
