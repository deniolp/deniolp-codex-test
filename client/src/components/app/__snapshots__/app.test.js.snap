// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div>
  <h3>
    Привет!
  </h3>
  <p>
    Эта программа рисует псевдографику, читая инструкции из файла /data/input.txt. Если вы сделали "npm i", а затем "npm run dev", вы уже видите это произведение искусства. Сами вы тоже можете рисовать с помощью подобных инструкций, написав их в "input.txt". "npm run dev" запускает 2 сервера (express на backend, а второй на клиенте). Нажатие на кнопку сохранит ваш рисунок в /data/output.txt
  </p>
  <button
    disabled={true}
    onClick={[Function]}
  >
    Save the painting
  </button>
  <textarea
    disabled="disabled"
    style={
      Object {
        "color": "green",
        "height": "75vh",
        "width": "100%",
      }
    }
    value=""
  />
</div>
`;
